application/models/LSActiveRecord.php

```diff
@@ -445,4 +445,33 @@ public function setEncryptedAttributeLabel(int $surveyId = 0, string $className,
         }
         
     }
+
+    /**
+     * Get the owner id of this record
+     * Used for Permission, to be extendable for each model with owner
+     * @return integer|null
+     */
+    public function getOwnerId()
+    {
+        return null;
```

Comment:
If not implemented, maybe throw exception instead?

Reply:
Now : it's a function for any Object. My opinion : no owner => return no owner

See usage here : https://github.com/LimeSurvey/LimeSurvey/pull/1632/files/2a87fd9fef2b1aa7f92e4fabeda3b99a226dd49e#diff-10077a3f8791500882779e35a827caf5e6578d5140625094d224de5e3c166780R547

Reply:
Where? Shows complete diff :joy: